lib/time.py

```diff
@@ -23,3 +55,39 @@ def sleep(secs):
 
 def time():
   return float(Now().UnixNano()) / Second
+
+
+_strftime_directive_map = {
+    'Y': '2006', 'y': '06', 'b': 'Jan', 'B': 'January', 'm': '01', 'd': '02',
+    'a': 'Mon', 'A': 'Monday',
+    'H': '15', 'I': '03', 'M': '04', 'S': '05', 'L': '.000',
+    'p': 'PM',
+    'Z':  'MST', 'z': '-0700',
+    '%': '%',
+}
+
+def strftime(format, tt=None): #pylint: disable=missing-docstring
+  t = (Unix(int(mktime(tt)), 0) if tt else Now()).Local()
+  ret = []
+  prev, n = 0, format.find('%', 0, -1)
+  while n != -1:
+    ret.append(format[prev:n])
+    next_ch = format[n+1]
+    if next_ch in 'cjUwWxX0123456789':
```

Comment:
@trotterdylan , I could reproduce that in my OSX, but linux is different.
Could you try on linux too?